corehq/apps/accounting/static/accounting/js/pricing_table.js

```diff
@@ -23,28 +24,30 @@ var PricingTable = function (options) {
 
     var self = {};
 
-    self.oCurrentPlan = ko.observable(options.currentPlan);
+    self.oCurrentPlan = ko.observable(options.currentEdition);
     self.oNextSubscription = ko.observable(options.nextSubscriptionEdition);
     self.oStartDateAfterMinimumSubscription = ko.observable(options.startDateAfterMinimum);
     self.oCurrentPrice = ko.observable(options.currentPrice);
     self.oIsPriceDiscounted = ko.observable(options.isPriceDiscounted);
+    self.currentIsAnnualPlan = options.currentIsAnnualPlan;
     self.editions = options.editions;
     self.isRenewal = options.isRenewal;
     self.subscriptionBelowMinimum = options.isSubscriptionBelowMin;
     self.invoicingContact = options.invoicingContact;
 
-    self.oSelectedPlan = ko.observable(options.currentPlan);
+    self.oSelectedPlan = ko.observable(options.currentEdition);
```

Comment:
rename to `oSelectedEdition`

Reply:
https://github.com/dimagi/commcare-hq/pull/36499/commits/8834204887f6d555f4e87d1df0e634bf8bf31304